source/includes/pipeline_config/_018-ant-task.md.erb

```diff
@@ -0,0 +1,26 @@
+<%
+json ='{
+  "tasks": [
+    {
+      "type": "ant",
+      "attributes": {
+        "run_if": [
+          "passed"
+        ],
+        "working_directory": "script/build",
+        "build_file": null,
+        "target": null
+      }
+    }
+  ]
+}'
+%>
+<%=
+describe_sub_object 'The ant task attributes', json: json do
+  run_if                       'Array',        'The run_if condition specifies when a task should be allowed to run.', one_of: %w(passed failed any)
+  build_file                   'String',       'The path to Ant build file.'
+  target                       'String',       'The ant target(s) to run.'
```

Comment:
@jyotisingh @arvindsv - I'm wondering if we should rename this attribute to `ant_args`, since this is really what it is, you could in theory pass in whatever ant accepts in here.

Ditto with some of the other tasks, rake, nant.

I'll submit a PR to make that change if you agree.